thirdparty/hnswlib/hnswlib/hnswalg.h

```diff
@@ -1392,7 +1392,7 @@ class HierarchicalNSW : public AlgorithmInterface<dist_t> {
         }
 
         std::unique_ptr<int8_t[]> query_data_sq;
-        const data_t* raw_data = (const data_t*)query_data;
+        [[gnu::unused]] const data_t* raw_data = (const data_t*)query_data;
```

Comment:
[[maybe_unused]]? Why use gnu directives? Clang may fail

Reply:
change to [[maybe_unused]]